persistence-cassandra/core/src/main/scala/com/lightbend/lagom/internal/persistence/cassandra/CassandraProvider.scala

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (C) 2016-2017 Lightbend Inc. <https://www.lightbend.com>
+ */
+package com.lightbend.lagom.internal.persistence.cassandra
+
+import akka.actor.ActorSystem
+
+/**
+ * Internal API
+ */
+private[lagom] class CassandraProvider(system: ActorSystem) {
```

Comment:
I think `SlickProvider` does load config settings and a ton more things. 😂 
I agree that `CassandraReadSideConfigProvider` is too long, that's a side effect of breaking the code in such small pieces.

We could have a `CassandraReadSideSettings` to continue with the naming introduced by APC where there's a `akka.persistence.cassandra.session.CassandraSessionSettings`.